lib/SILOptimizer/Mandatory/DIMemoryUseCollector.cpp

```diff
@@ -475,9 +475,14 @@ namespace {
 /// Gathers information about a specific address and its uses to determine
 /// definite initialization.
 class ElementUseCollector {
+public:
+  typedef SmallPtrSet<SILFunction *, 8> FunctionSet;
+
+private:
   SILModule &Module;
   const DIMemoryObjectInfo &TheMemory;
   DIElementUseInfo &UseInfo;
+  FunctionSet &visitedClosures;
```

Comment:
Nitpick: to be consistent with the other members this should be VisitedClosures

Reply:
done